arcticpathing/pathing.py

```diff
@@ -55,13 +55,26 @@ def generate_path_coords(node, start):
     return {
         'path': path[::-1],
         'path_coords': path_coords[::-1],
-        'distance': path_distance,
+        'path_distance': round(path_distance * constants.GRID_SIZE, constants.DATA_PRECISION),
     }
 
 
-def weight(node1, node2):
+def weight(node1: Node, node2: Node):
     c1 = node1.get_coords()
     c2 = node2.get_coords()
     t1 = data.get_thickness(c1)
     t2 = data.get_thickness(c2)
     return 2 ** (t1 + t2)
+
+
+def parse_string(path_string: str):
```

Comment:
I'd probably rename this to `parse_request_path_str` or something like that. Might also make more sense to live with the webapp code since it appears to be specifically used for that. In either case, a docstring here would be useful!